langchain/document_loaders/pdf.py

```diff
@@ -362,3 +363,26 @@ def load(self) -> List[Document]:
             contents = self.clean_pdf(contents)
         metadata = {"source": self.source, "file_path": self.source}
         return [Document(page_content=contents, metadata=metadata)]
+
+
+class PDFPlumberLoader(BasePDFLoader):
+    """Loader that uses pdfplumber to load PDF files."""
+
+    def __init__(self, file_path: str) -> None:
+        """Initialize with file path."""
+        try:
+            import pdfplumber  # noqa:F401
+        except ImportError:
+            raise ValueError(
+                "pdfplumber package not found, please install it with "
+                "`pip install pdfplumber`"
+            )
+
+        super().__init__(file_path)
+
+    def load(self, **kwargs: Optional[Any]) -> List[Document]:
```

Comment:
`load` should not take arguments or kwargs, so the interface remains `load()`

```suggestion
    def load(self) -> List[Document]:
```

All configuration differences get restricted to the __init__,  making it possible to substitute one loader for another in code that relies on loaders having a uniform load interface.